app/models/group.rb

```diff
@@ -153,4 +153,48 @@ def delete_permission(current_user, removed_user)
       errors.add(:delete_permission, "Unauthorized")
     end
   end
+
```

Comment:
Eventually we might want to make communities and subcommunities configurable somehow, but for now we are just trying to see if this new design works for PPPL.